easybuild/easyblocks/c/cuda.py

```diff
@@ -136,8 +137,15 @@ def create_wrapper(wrapper_name, wrapper_comp):
         for comp in (self.cfg['host_compilers'] or []):
             create_wrapper('nvcc_%s' % comp, comp)
 
+        # ldconfig is usually in /sbin or /usr/sbin
+        curr_path = os.environ.get('PATH', '')
+        setvar('PATH', os.pathsep.join(['/sbin', '/usr/sbin', curr_path]))
+        if not which('ldconfig'):
+            raise EasyBuildError("Unable to find the 'ldconfig' in PATH")
```

Comment:
Please mention `$PATH` in the error message, just `PATH` is a bit ambiguous

Reply:
You mean:
```
raise EasyBuildError("Unable to find the 'ldconfig' in $PATH")
 
```

Reply:
@boegel ping

Reply:
Yes, like that is better (although I guess bonus points can be earned if you mention the `$PATH` value as well in the error message.